catalogData/redis32/3x-ha/k8s/service-proxy.json

```diff
@@ -0,0 +1,33 @@
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "org": "$org",
+      "space": "$space",
+      "catalog_service_id": "$catalog_service_id",
+      "catalog_plan_id": "$catalog_plan_id",
+      "service_id": "$service_id",
+      "idx_and_short_serviceid": "$idx_and_short_serviceid",
+      "managed_by": "TAP"
+    },
+    "name": "$idx_and_short_serviceid-proxy",
+    "annotations": {
+      "service.beta.kubernetes.io/aws-load-balancer-internal": "0.0.0.0/0"
```

Comment:
I originally borrowed this from the `catalogData/redis28/standard/k8s/service.json`. I thought I needed it after reading [these docs](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer) for the proxy to talk to both CF and the Sentinels.

```sh
ack service.beta.kubernetes.io/aws-load-balancer-internal -- catalogData/redis28
catalogData/redis28/standard/k8s/service.json
7:      "service.beta.kubernetes.io/aws-load-balancer-internal": "0.0.0.0/0"
```